client/src/components/requester_task/partial-location.html

```diff
@@ -1,20 +1,29 @@
 <div id="requester-task-location">
-
-  <h1> Pick Shop to make a request </h1>
-  <div id="map">
-  </div>
-  <div id="shop" ng-repeat="shop in ctrl.activeShops">
-    <div ng-click= "ctrl.selectLocation(shop)">
-     Select Location: {{ shop }}
+  <div class="container">
+    <h1 class="requester-task-stage"> Pick Location </h1>
+    <div class="form-group">
+      <div class="row">
+        <div class="col-md-6" id="map"></div>
+        <div class="col-md-6" ng-repeat="shop in ctrl.activeShops">

```

Comment:
Putting the ng-repeat in this `<div>` will repeat everything inside. I think your intention was to create `<li>` for each activeshops. The way it's currently set up, it'll create `<ul>` as well. 
So something like ↓ will happen.
```
<ul>
  <li></li>
</ul>

<ul>
  <li></li>
</ul>

<ul>
  <li></li>
</ul>
```